src/components/LedgerTable.tsx

```
@@ -0,0 +1,27 @@
+interface LedgerTableProps {
+  data: Record<string, string>;
+}
+
+export const LedgerTable = (props: LedgerTableProps) => (
+  <table>
+    <thead>
+      <th>Key</th>
+      <th>Description</th>
+    </thead>
+    {Object.keys(props.data).map((key: string) => (
+      <TableRow value={key} description={props.data[key]} />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)